src/register-org/containers/registered-address/registered-address.component.ts

```diff
@@ -40,8 +71,95 @@ export class RegisteredAddressComponent extends RegisterComponent implements OnI
     this.router.navigate([this.registerOrgService.REGISTER_ORG_NEW_ROUTE, 'registered-address']);
   }
 
+  private isFormValid(): boolean {
+    let errorFound = false;
+    this.addressErrors = [];
+    if (this.startedInternational && this.isInternational === undefined) {
+      this.addressErrors.push({
+        id: 'govuk-radios',
+        message: AddressMessageEnum.NO_OPTION_SELECTED
+      });
+      errorFound = true;
+    } else if (this.formGroup.invalid) {
+      if (this.isControlInvalid('addressLine1')) {
+        this.addressErrors.push({
+          id: 'addressLine1',
+          message: AddressMessageEnum.NO_STREET_SELECTED
+        });
+      }
+      if (this.isControlInvalid('postTown')) {
+        this.addressErrors.push({
+          id: 'postTown',
+          message: AddressMessageEnum.NO_CITY_SELECTED
+        });
+      }
+      if (this.isControlInvalid('country') && this.isInternational) {
+        this.addressErrors.push({
+          id: 'country',
+          message: AddressMessageEnum.NO_COUNTRY_SELECTED
+        });
+      }
+      if (this.isControlInvalid('postCode') && !this.isInternational) {
+        this.addressErrors.push({
+          id: 'postCode',
+          message: AddressMessageEnum.NO_POSTCODE_SELECTED
+        });
+      }
+      errorFound = true;
+    }
+    if (errorFound) {
+      this.mainContentElement.nativeElement.scrollIntoView({ behavior: 'smooth' });
```

Comment:
@connorpgpmcelroy - scrolling to top is no longer required as it is automatically handled by `xuilib-hmcts-error-summary` component.